tardis/io/tests/data/tardis_configv1_verysimple.yml

```diff
@@ -4,8 +4,8 @@ supernova:
     luminosity_requested: 2.8e9 solLum
     time_explosion: 13 day
 
-atom_data: kurucz_atom_pure_simple.h5
-
+# atom_data: kurucz_atom_pure_simple.h5
+atom_data: kurucz_cd23_chianti_H_He.h5
```

Comment:
I changed that since i wasn't able to find the specified atom_data file and it seemed to not matter because this setting is overwritten when running the tests anyway.

It bight be a good idea to discuss what should happen.
For now i can revert that to the old setting